src/main/java/uk/gov/ons/ssdc/jobprocessor/config/AppConfig.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.ons.ssdc.jobprocessor.config;
+
+import com.godaddy.logging.LoggingConfigs;
+import com.google.cloud.spring.pubsub.core.PubSubTemplate;
+import com.google.cloud.spring.pubsub.support.PublisherFactory;
+import com.google.cloud.spring.pubsub.support.SubscriberFactory;
+import com.google.cloud.spring.pubsub.support.converter.JacksonPubSubMessageConverter;
+import java.util.TimeZone;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import uk.gov.ons.ssdc.jobprocessor.utility.ObjectMapperFactory;
+
+@Configuration
+@EnableScheduling
+public class AppConfig {
+  @Value("${spring.task.scheduling.pool.size}")
+  private int schedulingPoolSize;
+
+  @Bean
+  public PubSubTemplate pubSubTemplate(
+      PublisherFactory publisherFactory,
+      SubscriberFactory subscriberFactory,
+      JacksonPubSubMessageConverter jacksonPubSubMessageConverter) {
+    PubSubTemplate pubSubTemplate = new PubSubTemplate(publisherFactory, subscriberFactory);
+    pubSubTemplate.setMessageConverter(jacksonPubSubMessageConverter);
+    return pubSubTemplate;
+  }
+
+  @Bean
+  public JacksonPubSubMessageConverter messageConverter() {
+    return new JacksonPubSubMessageConverter(ObjectMapperFactory.objectMapper());
+  }
+
+  @Bean
+  public TaskScheduler taskScheduler() {
+    ThreadPoolTaskScheduler taskScheduler = new ThreadPoolTaskScheduler();
+    taskScheduler.setPoolSize(schedulingPoolSize);
+    return taskScheduler;
+  }
+
+  @PostConstruct
+  public void init() {
+    LoggingConfigs.setCurrent(LoggingConfigs.getCurrent().useJson());
```

Comment:
Add:
  @Value("$logging.profile")
  private String loggingProfile;

And change to 

  @PostConstruct
  public void init() {
    if (loggingProfile.equals("STRUCTURED")) {
      LoggingConfigs.setCurrent(LoggingConfigs.getCurrent().useJson());
    }

    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
  }